website/content/en/docs/getting-started/getting-started-with-karpenter/prometheus-values.yaml

```diff
@@ -13,7 +13,7 @@ extraScrapeConfigs: |
       - role: endpoints
         namespaces:
           names:
-          - kube-system
+          - $KARPENTER_NAMESPACE
```

Comment:
Thank you for opening this PR! Would you be willing add this changes to v0.32 and v0.33?

Reply:
Thank you, I've updated 0.32 and 0.33 as well now.